### PR TITLE
Add coverage check

### PR DESCRIPTION
**Pull request by okurz**
* t: Rely on done_testing for 02-test_ocr.t
* Make tests callable from any dir
* Rename tests to more common extension '.t'

### CODE REVIEW

File: doc/backend_vars.asciidoc [hunk 1]
```diff
@@ -29,15 +51,15 @@ Variable;Values allowed;Default value;Explanation
...
 BOOTFROM;chars;undef;Influences order of boot devices. See qemu -boot option
+BOOT_HDD_IMAGE;boolean;;enables boot from HDD_1 (BOOTFROM has higher priority)
 CDMODEL;see qemu -device ?;undef;Storage device for virtualized CD
 HDDFORMAT;;;
 HDDMODEL;see qemu -device ?;virtio-blk;Storage device for virtualized HDD.
 HDDSIZEGB;integer;10;Creates HDD with specified size in GiB
+HDDSIZEGB_$i;;;
 HDD_$i;filename;;Filename of HDD image to be used for VM. Up to 9
 ISO;filename;;Filename of ISO file to be attached to VM
-ISO_$i;filename;;Aditional ISO to be attached to VM. Up to 9
```
Comment (dzedro) [inline doc/backend_vars.asciidoc]: Why ? we don't use ISO_* anymore ?

Comment (okurz) [inline doc/backend_vars.asciidoc]: I hate this frikin' broken backend vars "test"

**Comment by coolo** [comment]
merge conflict - and unless you invest heavily in a test framework to test the multi process setup I will decrease your limit to pass my PRs (and merge every other PR that does so). Because the coverage is so low for a reason - it takes more than unit tests to test this beast for real ;(

**Comment by okurz** [comment]
> merge conflict

obviously from your recent changes that came afterwards, updated.

> and unless you invest heavily in a test framework to test the multi process setup I will decrease your limit to pass my PRs (and merge every other PR that does so). Because the coverage is so low for a reason - it takes more than unit tests to test this beast for real ;(

That's ok, we have to start somewhere and we already improved the tests over the recent months. Explicitly lowering the threshold is better than not knowing that the coverage decreased.

**Pull request merged**